### PR TITLE
fix: removes nodeModules from curBackend

### PR DESCRIPTION
**Pull request by akshbhu**
*Description of changes:*

Removing NodeModules folder from the #currentcloudbackend

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-e2e-core/src/categories/lambda-function.ts [hunk 1]
```diff
@@ -472,3 +474,12 @@ const getRuntimeDisplayName = (runtime: FunctionRuntimes) => {
       throw new Error(`Invalid runtime value: ${runtime}`);
   }
 };
+
+export function validateRemovalNodeModulesDir(projRoot) {
```
Comment (jhockett) [inline packages/amplify-e2e-core/src/categories/lambda-function.ts:478]: nit: `validateNodeModulesDirRemoval` is a little easier to read

File: packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import * as fs from 'fs-extra';
 import * as path from 'path';
 import { hashElement } from 'folder-hash';
+import glob from 'glob';
```
[the remark this answers is not in the document]
Comment (akshbhu) [reply packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts:4]: nice catch !!

File: packages/amplify-e2e-core/src/categories/lambda-function.ts (hunk 1 above)
Comment (akshbhu) [reply packages/amplify-e2e-core/src/categories/lambda-function.ts]: done!!

File: packages/amplify-cli/package.json [hunk 3]
```diff
@@ -96,7 +96,8 @@
     "tar-fs": "^2.1.1",
     "update-notifier": "^4.1.0",
     "which": "^2.0.2",
-    "winston": "^3.3.3"
+    "winston": "^3.3.3",
+    "glob": "^7.1.6"
```
Comment (edwardfoyle) [inline packages/amplify-cli/package.json:100]: nit: sort imports alphabetically

Comment (akshbhu) [reply packages/amplify-cli/package.json]: done!!

**Review by jhockett — APPROVED** [review]
LGTM!

**Pull request merged**